src/components/templates/settingsLayout.tsx

```diff
@@ -7,13 +7,13 @@ import { LogoCatLarge, PageTitle } from "@components/atoms";
 import { Sidebar, TitleTopbar } from "@components/organisms";
 import { SettingsMenu } from "@components/organisms/settings";
 
-export const SettingsLayout = () => {
+export const SettingsLayout = ({ variant }: { variant: "userSettings" | "organization" }) => {
 	const { t: tSettings } = useTranslation("settings", { keyPrefix: "topbar" });
 	const { t } = useTranslation("global", { keyPrefix: "pageTitles" });
 	const [pageTitle, setPageTitle] = useState<string>(t("base"));
 	const { pathname } = useLocation();
 
-	const topbarTitle = pathname.startsWith("/settings") ? tSettings("personalSettings") : tSettings("title");
+	const topbarTitle = pathname.startsWith("/settings") ? tSettings("personalSettings") : tSettings("organization");
```

Comment:
Here we can work with `variant` instead of `pathname.startsWith` ?

Reply:
changed